### DISCUSSION TITLE
the forecast task failed when increasing forecast hour

### DISCUSSION THREAD

**Discussion by DanWu85** [hafs-community/HAFS#195] [General]
Hello,

I'm running a 5-day long experiment. In order to reduce the computational cost, the forecast hour is set to 12 initially. Now I'm trying to rerun the forecast task of the last cycle and extent its forecast hour to 126. However, this task failed due to time limit.

The walltime of the forecast task in my configuration is 07:59:00.
The current resources used for forecast tasks are:
<!ENTITY FORECAST_RESOURCES_1080PE "<nodes>54:ppn=20:tpp=2</nodes><envar><name>TOTAL_TASKS</name><value>1080</value></envar><envar><name>NCTSK</name><value>20</value></envar>&FORECAST_EXTRA;"> 

I also tried a larger PE
<!ENTITY FORECAST_RESOURCES_1380PE "<nodes>69:ppn=20:tpp=2</nodes><envar><name>TOTAL_TASKS</name><value>1380</value></envar><envar><name>NCTSK</name><value>20</value></envar>&FORECAST_EXTRA;">

However, it still cannot complete within the walltime. Does anyone know how to solve this issue? Many thanks!

-------------------------

Update: The reason for the unfinished forecast task is caused by the huge domain and high resolution in my configuration. 

I've increased the forecast PE from 1080 to 1980. However, still didn't see much progress. Does anyone knows how to make the calculation more efficient?  Many thanks!

Dan

**Comment by BinLiu-NOAA** [comment]
Issue obsolete/fixed.
